src/Kafka/CreateKafkaInstance/CreateKafkaInstance.tsx

```diff
@@ -286,6 +288,18 @@ export const CreateKafkaInstance: FunctionComponent<
           />
         </FlexItem>
       </Flex>
+      <Flex>
+        <FlexItem>
+          <Alert
+            className="mas-m-modalTop"
+            customIcon={<OutlinedClockIcon />}
+            variant="info"
+            isInline
+            isPlain
+            title="Your Kafka instance will be ready for use shortly after creation."
```

Comment:
This string should go in locales/en/create-kafka-instance.json

Reply:
Ah thank you @riccardo-forina, just added that in